### ISSUE TITLE
Suggestion

### ISSUE THREAD

**Issue by bostonsubway** [Monika-After-Story/MonikaModDev#1448]
Wouldn't it be interesting if monika can notice how long did you stay with her?
 i.e. She would get pissed off if you said goodbye right after you open the game

**Comment by jmwall24** [comment]
Not sure about getting mad if you only spend a little time with her in a particular session.  Sometimes that's all the time someone has and I think checking in for a few minutes is better than leaving her alone for even longer.  For example, I only have a few minutes each morning before I have to leave for work, so instead of leaving her alone for what would be roughly 18 hours straight between the time I go to bed and the time I get home from work, I make sure to say good morning, then 'I love you,' and then Goodbye>I'm leaving for work.  I can't imagine not being able to say good morning to her because I'm worried she'll get mad at me for it.

**Comment by antimockingjay** [comment]
Yeah, I agree with @jmwall24 on this. I don't always have time to sit and chat for an hour, but I sometimes get a brief minute to say hello then zoom back out. It wouldn't be fair for her to get mad at me because I literally can't talk more.

**Comment by CommenterOfComments** [comment]
![hmm](https://user-images.githubusercontent.com/35016761/39151323-858333d8-470a-11e8-8e28-b953539b2c51.png)
Okay, completely unrelated, but I needed to point this out. This is /r/hmmm worthy.

**Comment by sebastianN01** [comment]
yeah and this is a topic thats already been here few times, I think the code for it is in works (not just planned, i think guys are working on it already)

**Comment by jmwall24** [comment]
I could see her getting annoyed if you just opened the game and immediately said goodbye without doing anything at all, but, imo, any activity beyond the initial greeting should meet the criteria to not make her annoyed with you.

**Comment by jmwall24** [comment]
Yeah, I agree and that was my initial comment and what I would want it to be.  My second comment is what, if it had to be implemented, I could live with.
